### PR TITLE
feat: add explanation for cryptographic use in all charms

### PR DESCRIPTION
**Pull request by morphis**

### CODE REVIEW

File: explanation/cryptography/crypto_charms.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+* [`anbox-cloud-dashboard`](https://charmhub.io/anbox-cloud-dashboard)
+* [`lxd-integrator`](https://charmhub.io/lxd-integrator)
+
+The charms for Anbox Cloud make use of cryptographic technology for:
+
+* Creation of TLS certificates
+
+## Creation of TLS certificates
+
+When Anbox Cloud is deployed without the use of an external CA, the charms will generated self-signed certificates using the [cryptography`](https://pypi.org/project/cryptography/) Python package. The private key used for signing has a size of 4096 bits.
```
Comment (jat-canonical) [inline explanation/cryptography/crypto_charms.md:20]:
```suggestion
When Anbox Cloud is deployed without the use of an external CA, the charms will generated self-signed certificates using the [cryptography](https://pypi.org/project/cryptography/) Python package. The private key used for signing has a size of 4096 bits.
```

**Review by jat-canonical — APPROVED** [review]
one small fix otherwise. LGTM .

**Review by keirthana — CHANGES_REQUESTED** [review]
Minor suggestions

File: explanation/cryptography/crypto_charms.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Packages used
+
+* [cryptography from Pypi](https://pypi.org/project/cryptography/)
```
Comment (keirthana) [inline explanation/cryptography/crypto_charms.md:24]: Can you add Pypi to the wordlist to pass spellcheck?

**Review by keirthana — APPROVED** [review]
Thanks, LGTM

**Review by ajanon — APPROVED** [review]
LGTM

**Pull request merged**